.devcontainer/docker-compose-local.yaml

```diff
@@ -9,6 +9,7 @@ services:
     ports:
       - 5432:5432
     environment:
+      # https://hub.docker.com/_/postgres
```

Comment:
I'm not sure we need this comment since folks find the docker image here https://github.com/mozilla-ai/lumigator/blob/caa5b80ea8c982264f28fadc3bdabb7aa7c71bad/.devcontainer/docker-compose-local.yaml#L5C9-L5C30

Reply:
Always a fan of docs, but I think this is redundant

Reply:
I removed all comments in docker compose.